### ISSUE TITLE
Make the web app report adhere to the new `severeIssueThreshold` configuration

### ISSUE THREAD

**Issue by sschuberth** [oss-review-toolkit/ort#4453]
@tsteenbe would you be able to help with resolving the final bit, and make the web app report adhere to the new `severeIssueThreshold` configuration in its issue counts in the tabs?

_Originally posted by @sschuberth in https://github.com/oss-review-toolkit/ort/issues/4096#issuecomment-856000271_

**Comment by tsteenbe** [comment]
@sschuberth  This issue does not include full description of the change so just to confirm - "You want if severeIssueThreshold is set to Warning that the WebApp only shows warnings and higher severity correct?"

**Comment by sschuberth** [comment]
I agree this issue needs refinement. I'll try to come up with a mock-up as IMO more UI changes are required to nicely display what we're interested in.

**Comment by sschuberth** [comment]
This issue was briefly discussed as part of backlog grooming. The `severeIssueThreshold` was originally introduced for the CLI tools (and their resulting exit status) to be able to configure from what severity on an issue counts as a "failure that needs to be addressed" (as not everyone want to invest time to address warnings, for example).

However, when viewing reports there is no "exit status". So the proposal is to either
- just use the `severeIssueThreshold` as the default filter for issues, to only show issues that exceed the threshold and need to be addressed, or
- to add something to the summary that say whether according to the threshold the contained issues are acceptable or not.
